snapcraft/internal/pluginhandler/__init__.py

```diff
@@ -29,7 +29,15 @@
 
 import snapcraft.extractors
 from snapcraft import file_utils
-from snapcraft.internal import common, elf, errors, repo, sources, states
+from snapcraft.internal import (
+    common,
+    elf,
+    errors,
+    repo,
+    sources,
+    states,
+    steps,
+)
```

Comment:
this import style really drives me nuts for some reason I cannot explain :stuck_out_tongue:

Reply:
Haha, this one was long enough I had to split it, sorry!

Reply:
If you use autopep8 it will use the format I like :-P